### ISSUE TITLE
Voicemail notification always on

### ISSUE THREAD

**Issue by commodore64guy** [GrapheneOS/os-issue-tracker#2633]
I have recently noticed that after retrieving and deleting voicemails, the notification dot on the phone icon does not turn off. This seems to have started a couple updates ago...literally in the past month.  Restarting my phone used to clear the message indicator but not anymore. My lock screen allows says "Phone" with the recording icon next to it.  Now my phone constantly indicates I have a voicemail when I don't.

When this started, I also noticed I could no longer press and hold the dot on the phone icon to dial voicemail. Also, doing a down swipe from the main menu and calling my voicemail that way, still does not clear the notification dot or the message in notifications. I have isolated this issue to only occurring when a voicemail is left for me. When I miss a call, the dot appears as it should, and when I open the phone to review the missed call, the dot goes away. The missed call message in notifications goes away also. The notification dot works appropriately with text messages.
 
I have a Pixel 6 Pro running GrapheneOS Android version 14, updated October 6, 2023. Build number UP1A.231005.007.2023102300

If it helps, I'm also running;

Simple Mobile Tools calendar, music player, voice recorder
- Magic Earth
- Simple Mobile Tools Caledar
- Simple Mobile Tools Music Player
- NewPipe
- Farmerbb Notepad
- rhvoice
- Noctua Software Stellarium
- Wangdaye Geometric Weather

All the above apps were installed within days of laying down the OS and everything has been stable.

I'm offering this info to help notify the awesome folks at GrapheneOS about this issue in hopes of a future fix.

Please let me know if there is anything I can do to help resolve this...logs, sequence of events, additional details.  Happy to help with my somewhat limited knowledge and be a part of the solution.  Thanks in advance for your help and I hope we can resolve this.

**Comment by rmdaw00** [comment]
Can confirm same phone Pixel 6 Pro. UP1A.231005.007.2023102300 same issue

**Comment by commodore64guy** [comment]
UPDATE - I contacted Verizon again and after about 45 minutes and several tests, they seem to have resolved this issue.

First, they said they "corrected" some network settings on their end. They could not provide any specifics regarding those corrections. They asked me to shut down my phone for 5 minutes at which time they would call me back and leave a message...which they did. Notifications came on but would not turn off after deleting the message.

The 2nd thing they did was they removed my voicemail account and reapplied it (their words). They left a message again with no improvements. However, they then asked me to just restart the phone and they would again call and leave a message. That time it worked. They even called me a 2nd time and my notification came on immediately and went off immediately after deleting the voicemail.

I have also tested this by having a friend leave a voicemail and notifications seem to be working properly again.

I'll provide what will hopefully be a final follow up to this in 1 week at which time I'll be happy to close this if I can.  (I do see an option at the bottom for closing this).

Hope this helps if anything else is having the same issue.

**Comment by thestinger** [comment]
Unclear if this is an OS issue or a carrier issue.

@rmdaw00 Which carrier do you have this issue with?

**Comment by commodore64guy** [comment]
@thestinger I wish I had paid closer attention to exactly when this came about for me.  I do know it was right after a recent update.  Is it possible the update caused Verizon to no longer recognize my phone to send the proper signal to turn off my notification?  (just a thought).

Also, if it helps, I also noticed when I received a voicemail, and the dot notification appeared on my home screen phone icon, I could press and hold the dot and it would provide an option to call voicemail right from there.  That doesn't seem to work anymore.  When I do that, App info does appear as expected.  Obviously, being able to press the dot and call voicemail from there is not at all a big deal...at least not for me ;)

I know you were looking for a reply from rmdaw00 and I hope he responds, but I thought I'd add this in hopes it helps you. 

Thanks for this awesome phone OS!!!  Kinda wish I was a developer and not a Microsoft guy ;)  It would be cool to be involved with GrapheneOS.

**Comment by thestinger** [comment]
Android 14 removed the notifications from the launcher. It's intended.

**Comment by commodore64guy** [comment]
Ah ha! Got it...thanks, I won't troubleshoot that anymore.

**Comment by thestinger** [comment]
Other launchers could still do it but AOSP Launcher and the Pixel Launcher fork of it removed it. Most other launchers are forks of AOSP launcher but some of them entirely diverge from it and don't take the new changes.

**Comment by thestinger** [comment]
I think it was removed because notifications are becoming more and more capable, but the ones included in the launcher were already quite a bad UI barely showing any of the info, etc. and it's not really a good experience compared to using the notification drawer. It's unfortunate they ever added it and got people into the habit of using it since change is painful.

**Comment by commodore64guy** [comment]
LOL at "change is painful."  "Who Moved My Cheese" published 1988 by Spencer Johnson.

Are the specific changes made in an update posted anywhere?  What I've seen is above my understanding.  Are changes posted in a scaled down Joe The Plumber explanation anywhere?  I shouldn't say this, but even as an IT guy, I don't understand some of the update postings.

**Comment by thestinger** [comment]
This change is part of Android 14. https://grapheneos.org/releases#2023100800 was our first release based on it.

> rebased onto UP1A.231005.007 Android Open Source Project release as the initial port of all GrapheneOS features to Android 14

We didn't try to document the Android 14 changes ourselves. There are many thousands of changes, including significant user facing changes. There's extensive Android documentation covering changes impacting app developers but this isn't an example of one of those since it's a user-facing UI change to the launcher which isn't part of what they'd cover in generic Android 14 release notes. They'd cover it in Pixel-specific release notes for the Pixel Launcher based on AOSP Launcher.

**Comment by commodore64guy** [comment]
Ouch...thousands of changes.  Yeah, I didn't think about all the behind-the-scenes things you guys work on that a user wouldn't notice.

Thanks for the link...I'll check it out.  And thanks for what you do!  I've had a flip phone since my first razr and have waited for something like GOS before making the leap to a quote unquote, smart phone.  Very happy with my Pixel.

Thanks stinger!

**Comment by thestinger** [comment]
> Ouch...thousands of changes. Yeah, I didn't think about all the behind-the-scenes things you guys work on that a user wouldn't notice.

Talking about the upstream Android Open Source Project changes. There are thousands of changes even just within certain specific repositories and there are over 500 repositories.

**Comment by commodore64guy** [comment]
WOW...just did some reading.  All I'll say, and I'm sure you'll understand, is that I hope you're doing better!!!

2 rules of life...
Rule 1 is to take care of number 1 (that is one's self)
Rule 2 is to make sure rule 1 was taken care of

Thanks again!!

**Comment by thestinger** [comment]
Not quite sure what you mean but would appreciate if you would contact me via @strcat:grapheneos.org on Matrix to discuss it rather than discussing it here.

**Comment by thestinger** [comment]
It doesn't support private messages yet.

**Comment by commodore64guy** [comment]
Closing this issue as resolved.

I contacted my carrier, Verizon, and after about 45 minutes and several tests, they seem to have resolved this issue.

First, they said they "corrected" some network settings on their end. They could not provide any specifics regarding those corrections. They asked me to shut down my phone for 5 minutes at which time they would call me back and leave a message...which they did. Notifications came on but would not turn off after deleting the message.

The 2nd thing they did was they removed my voicemail account and reapplied it (their words). They left a message again with no improvements. However, they then asked me to just restart the phone and they would again call and leave a message. That time it worked. They even called me a 2nd time and my notification came on immediately and went off immediately after deleting the voicemail.

I have also tested this by having a friend leave a voicemail and notifications seem to be working properly again.
